subproviders/hooked-wallet-ethtx.js

```diff
@@ -32,7 +32,7 @@ function HookedWalletEthTxSubprovider(opts) {
     opts.getPrivateKey(txData.from, function(err, privateKey) {
       if (err) return cb(err)
 
-      var tx = new EthTx(txData)
+      var tx = new Transaction(txData)
```

Comment:
why is this not TransactionData.fromTxData?

Reply:
i just looked at it would appear: no reason :sweat_smile: